--- slmdm-2.6.16/usb_st7554.c.orig	Mon Jun 17 15:57:00 2002
+++ slmdm-2.6.16/usb_st7554.c	Fri Dec 20 12:28:05 2002
@@ -160,7 +160,7 @@
 	unsigned int ctrl_ep; /* control endpoint */
 	u16 gpio_out;         /* gpio_out register map */
 	u32 intr_status;      /* interrupt status */
-	urb_t intr_urb;       /* interrupt urb */
+	struct urb intr_urb;  /* interrupt urb */
 
 	struct modem *modem;  /* modem high level instance */
 
@@ -484,7 +484,7 @@
 
 /* --------------------------------------------------------------------- */
 
-static void st7554_interrupt(urb_t *urb)
+static void st7554_interrupt(struct urb *urb)
 {
 	struct st7554_modem_state *s = urb->context;
 	u32 *status = urb->transfer_buffer;
@@ -534,7 +534,7 @@
 {
 	struct st7554_modem_state *s = u->context;
 	struct dmabuf *db = &s->mo.dma;
-	iso_packet_descriptor_t *p;
+	struct iso_packet_descriptor *p;
 	unsigned long flags;
 	int i;
 
@@ -621,10 +621,10 @@
 
 /* ----------------------------------------------------------------------- */
 
-static void mi_complete(urb_t *u)
+static void mi_complete(struct urb *u)
 {
 	struct st7554_modem_state *s = u->context;
-	iso_packet_descriptor_t *p;
+	struct iso_packet_descriptor *p;
 	unsigned long flags;
 	int i;
 
@@ -664,10 +664,10 @@
 }
 
 
-static void mi_setup_complete(urb_t *u)
+static void mi_setup_complete(struct urb *u)
 {
 	struct st7554_modem_state *s = u->context;
-	iso_packet_descriptor_t *p;
+	struct iso_packet_descriptor *p;
 	unsigned long flags;
 	int i;
 
